apps/web/src/pages/admin/transactions.tsx

```diff
@@ -20,7 +20,7 @@ export default function AdminTransactionsPage() {
       try {
         const claims = await adminService.getLoggedInUserPermissions()
         // If there is no admin role, throw error
-        if (!claims.roles || !claims.roles.includes(FirebaseClaim.admin)) {
+        if (!claims.claims || !claims.claims.includes(FirebaseClaim.admin)) {
```

Comment:
This is a little cleaner and doesn't involve iteration
```ts
const { claims } = await adminService.getLoggedInUserPermissions()
if (!claims?.[FirebaseClaim.admin]) { ... }
```

Reply:
Good call, just added in 59f3195.

Reply:
Ah, just realized that claims here don't have the same structure as we have on the backend. I was thinking claims.claims was an object. Anyway, this all seems good to merge at this point to me. We should talk about what's next.